test/tests/bcs/vectorpostprocessor/vectorpostprocessor.i

```diff
@@ -1,8 +1,3 @@
-Nu = 4
-k = 1
-half_channel_length = 0.5
-h = ${/ ${* ${Nu} ${k}} ${half_channel_length}}
```

Comment:
not supported anymore? :-/

Reply:
plain substitutions are still supported, but we decided we want to be deliberate and careful about introducing and potentially redesigning nifty functionality like this.  So out for now.